### PR TITLE
Feat/title support

### PR DESCRIPTION
**Pull request by cjpillsbury**
Adds title UI support, including css vars/parts, for mux-player and mux-player-react.

### CODE REVIEW

File: packages/mux-player/src/media-theme-mux/media-theme-mux.ts [hunk 1]
```diff
@@ -85,7 +88,6 @@ export default class MediaThemeMux extends MediaTheme {
           exportparts="layer, media-layer, poster-layer, vertical-layer, centered-layer"
         >
           <slot name="media" slot="media"></slot>
-          <media-poster-image slot="poster" src="${props.poster}"></media-poster-image>
```
Comment (luwes) [inline packages/mux-player/src/media-theme-mux/media-theme-mux.ts:88]: blocking: shouldn't be removed right?

**Review by luwes — CHANGES_REQUESTED** [review]

**Comment by luwes** [comment]
blocking: default font-size should be 16px from designs

File: packages/mux-player/src/media-theme-mux/media-theme-mux.ts (hunk 1 above)
Comment (cjpillsbury) [reply packages/mux-player/src/media-theme-mux/media-theme-mux.ts:88]: oh man good catch. Not sure why that happened with the rebase merge.

**Comment by luwes** [comment]
non-blocking: to me here the addition of the pipe char is less good looking I think. mostly because the spacing is asymmetrical 

SCR-20220830-q05

**Review by luwes — APPROVED** [review]
LGTM 👍

**Pull request merged**